go/mysql/flavor.go

```diff
@@ -424,6 +469,8 @@ func parseReplicationStatus(fields map[string]string) ReplicationStatus {
 	}
 	parseUint, _ = strconv.ParseUint(fields["Master_Server_Id"], 10, 0)
 	status.SourceServerID = uint(parseUint)
+	parseUint, _ = strconv.ParseUint(fields["SQL_Delay"], 10, 0)
+	status.SQLDelay = uint(parseUint)
```

Comment:
How are we using SQL_Delay? Can we assume it is never set?

Reply:
Same thing for Replication filters. Do we expect those to be ever set on Vitess-managed MySQL databases?
And if Vitess is not managing them, then we don't care anyway.

Reply:
We do not expect them to be set, but if they are then it could lead to unexpected failures in other components of Vitess. For example, if someone sets the SQL delay to 1 minute (for whatever imaginable reason) on a tablet, the ERS will keep failing over and over on trying to wait for the tablet to apply its relay log. It just wouldn't until the timeout (30 seconds) expires. Orchestrator used to track them and showed them on the UI.

It is easier to have the fields if we need them and remove them if we don't instead of not having them and needing them later on. So for now, I have elected to add all the fields that VTOrc tracks.

Also, we might want to add a failure scenario where we set SQL delay to 0 if not already. At the very least, ERS can benefit from it, ignoring the tablets that have some SQL delay setup, since they don't seem like viable primary candidates

Reply:
OK, let's err on the side of having more fields than fewer fields.